md/bpm-api.md

```diff
@@ -481,6 +481,19 @@ Retrieve humanTask objects that match the specified filters.
   `GET`
 * **Data Params**  
   [Standard search parameters](rest-api-overview.md#resource_search) are available.  
+  
+  * o: only parameters available in HumanTaskInstanceSearchDescriptor.java and in the result are accepted: 
```

Comment:
Use [HumanTaskInstanceSearchDescriptor](http://documentation.bonitasoft.com/javadoc/api/7.3/index.html) instead of HumanTaskInstanceSearchDescriptor.java